README-ar.md

```diff
@@ -0,0 +1,528 @@
+🌍
+*[Čeština](README-cs.md) ∙ [English](README.md) ∙ [Español](README-es.md) ∙ [Italiano](README-it.md) ∙ [日本語](README-ja.md) ∙ [한국어](README-ko.md) ∙ [Português](README-pt.md) ∙ [Русский](README-ru.md) ∙ [Slovenščina](README-sl.md) ∙ [Українська](README-uk.md) ∙ [中文](README-zh.md) ∙ [العربية](README-ar.md)*
+
+
+#  فنون أوامر وايعازات لنكس
+
+[![Join the chat at https://gitter.im/jlevy/the-art-of-command-line](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/jlevy/the-art-of-command-line?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+- [المقدمة](#meta)
```

Comment:
@0bserver07  @jlevy  You're right. I fixed the ancherrs.
The url will look ugly thu, cause it'll convert arabic to unicode so it will look something like this: 20%BC%50%...